Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -177,7 +177,6 @@ def update_learner(self):
             self.learner.use_default_preprocessors = True
         if self.learner is not None:
             self.learner.name = self.learner_name
-        self.learner.use_default_preprocessors = True
```

Comment:
Wasn't this supposed to be [enabled in widgets](https://github.com/biolab/orange3/pull/1657#issuecomment-263121263)?